centaur/src/main/resources/standardTestCases/workbench_health_monitor_check/workbench_health_monitor_check.wdl

```diff
@@ -9,8 +9,16 @@ workflow workbench_health_monitor_check {
 
 task run {
     command {
+        # Try to apt-get install jq and curl if they are missing. These should already be present in Travis so this would be a noop.
+        # Do this all quietly with output redirected to /dev/null since the logic below reads stdout.
+        (which jq > /dev/null && which curl > /dev/null) || (apt-get update > /dev/null && apt-get install -y jq curl > /dev/null)
```

Comment:
Yeah the existing code was subtly yuck, this makes it more explicitly so. 🙂 

Since the task explicitly uses the `Local` backend without a specified Docker image, it was previously assumed that both `jq` and `curl` were already present in the same environment in which Cromwell itself was running. For our local machines and Travis that appears to be generally true. However for the Cromwell Docker image this is not true since that contains `curl` but not `jq` (what's a Docker image without `curl`?).